graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -56,6 +49,13 @@
 its fields) removed. If "Dog" also appeared as a field in the schema's root type, it would be
 removed there as well.
 
+For both 1-1 or 1-many renaming, renamings only apply to types, fields, and enum values that exist
```

Comment:
Comment from 962: 

This gets moved because right now the comment's structure goes 
- 1-1 renaming description
- 1-many renaming description
- This paragraph
- 1-0 renaming description
- other information including implementation details

This PR moves the paragraph to go with existing implementation details, especially since #959 introduces more of these details.